common/test/acceptance/tests/lms/test_progress_page.py

```diff
@@ -240,7 +241,7 @@ def test_visibility_change_does_affect_score(self):
             self.assertEqual(self._get_problem_scores(), [(1, 1), (0, 1)])
             self.assertEqual(self._get_section_score(), (1, 2))
 
-    def test_progress_page_updates_when_student_state_deleted(self):
+    def test_delete_student_state_affects_score(self):
```

Comment:
Note: Although there's an acceptance test that makes sure the user's grade is updated accordingly after "Delete State", I didn't add one for the 2 "Rescore" buttons since those are acted upon asynchronously.

Instead, the end-to-end of the Rescore functionality is testing in a python integration test.